src/merge_ska_dict.rs

```diff
@@ -261,6 +263,7 @@ fn parallel_append<IntT>(
     k: usize,
     rc: bool,
     qual: &QualOpts,
+    max_reads: Option<usize>,
```

Comment:
Add the ignore too many arguments option for clippy here to avoid the warning below